.github/workflows/release.yml

```diff
@@ -20,3 +27,17 @@ jobs:
       go-version: 1.18
       config-file: .github/config-release.yml
       compile-builder: true
+
+  # In case this fails, e.g. build configuration changes, file an issue in slsa-verifier
+  if-failed:
+    runs-on: ubuntu-latest
+    needs: [builder]
+    if: always() && needs.builder.result == 'failure'
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
```

Comment:
nit: either remove the hash or use comment `# tag=v2.4.0` so that renovatebot can send us updates